src/samplers/libcall/stat.rs

```diff
@@ -0,0 +1,24 @@
+use rustcommon_metrics::{AtomicU32, AtomicU64, Source, Statistic};
+
+#[derive(Clone, Debug, Eq, PartialEq, Hash)]
+pub struct LibCallStatistic {
+    stat_path: String,
+}
+
+impl LibCallStatistic {
+    pub fn new(library: &str, func: &str) -> Self {
+        Self {
+            stat_path: format!("{}/{}", library, func),
```

Comment:
I'm wondering if we should prefix these at all - I'm trying to imagine how this would look mixed in with host-level stats that are namespaced by component. If the intent is to run just the single sampler as a service-specific sampler it's maybe less necessary to have a namespace prefix.

Reply:
I've added a namespace prefix: 'usercall' to all the metrics created by this sampler.